Patch by Robert Scheck <robert@fedoraproject.org> for libisofs >= 0.6.16, which solves the
multilib/multiarch problem by removing the timestamp within the footer from the doxygen
generated files.

--- libisofs-0.6.16/doc/doxygen.conf.in				2008-11-25 11:00:11.000000000 +0100
+++ libisofs-0.6.16/doc/doxygen.conf.in.multilib		2009-03-17 22:17:03.000000000 +0100
@@ -694,7 +694,7 @@
 # each generated HTML page. If it is left blank doxygen will generate a 
 # standard footer.
 
-HTML_FOOTER            = 
+HTML_FOOTER            = doc/footer_no_timestamp.html
 
 # The HTML_STYLESHEET tag can be used to specify a user-defined cascading 
 # style sheet that is used by each HTML page. It can be used to 
--- libisofs-0.6.16/doc/footer_no_timestamp.html		1970-01-01 01:00:00.000000000 +0100
+++ libisofs-0.6.16/doc/footer_no_timestamp.html.multilib	2009-03-17 22:18:47.000000000 +0100
@@ -0,0 +1,5 @@
+<hr size="1"><address style="text-align: right;"><small>Generated for $projectname by&nbsp;
+<a href="http://www.doxygen.org/index.html">
+<img src="doxygen.png" alt="doxygen" align="middle" border="0"></a> $doxygenversion </small></address>
+</body>
+</html>
